SSLeay.xs

```diff
@@ -4171,7 +4171,6 @@ P_X509_get_crl_distribution_points(cert)
         int i, j;
     PPCODE:
         points = X509_get_ext_d2i(cert, NID_crl_distribution_points, NULL, NULL);
-        if (points)
```

Comment:
Not related to the topic of the pull request, but I added one more change: The `if` statement seems to be unnecessary because `sk_*_num()` correctly works if NULL is passed to it. The missing indentation helps here: the body of the `if` statement was the `for` loop which is no longer conditional. However, we don't even need to reindent it.